client/src/test/java/io/split/engine/matchers/EqualToSemverMatcherTest.java

```diff
@@ -0,0 +1,22 @@
+package io.split.engine.matchers;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests for EqualToSemverMatcher
+ */
+
+public class EqualToSemverMatcherTest {
+
+    @Test
+    public void works() {
+        EqualToSemverMatcher equalToSemverMatcher = new EqualToSemverMatcher("2.1.8");
+
+        assertTrue( equalToSemverMatcher.match("2.1.8", null, null, null) == true);
```

Comment:
should be: `assertTrue(equalToSemverMatcher.match("2.1.8", null, null, null));`

Reply:
will update